### PR TITLE
fix: remove nested ternary operator and simplify ping peer functionality

### LINKED ISSUE #57: Eliminate nested ternary operators
Right now there is a number of nested ternary operators in several components. All of them have a `// FIXME` comment and should be removed.

### PR DESCRIPTION
**Pull request by vojtechsimetka**
Resolves #57

### CODE REVIEW

File: src/pages/peers/PeerTable.tsx [hunk 1]
```diff
@@ -27,28 +27,41 @@ interface Props {
...
     beeDebugApi
       ?.pingPeer(peerId)
       .then(res => {
-        setPeerLatency([...peerLatency, { peerId: peerId, rtt: res.rtt, loading: false }])
+        setPeerLatency({ ...peerLatency, [peerId]: { rtt: res.rtt, loading: false } })
       })
       .catch(() => {
-        setPeerLatency([...peerLatency, { peerId: peerId, rtt: 'error', loading: false }])
+        setPeerLatency({ ...peerLatency, [peerId]: { rtt: 'error', loading: false } })
       })
```
Comment (Cafe137) [inline src/pages/peers/PeerTable.tsx:58]: Hmm I am wondering if there is a race condition here. Not sure, but I'd probably use the arrow function for `setPeerLatency` which provides the previous value.

File: src/pages/peers/PeerTable.tsx [hunk 2]
```diff
@@ -27,28 +27,41 @@ interface Props {
   peers: Peer[] | null
 }
 
+interface PeerLatency {
+  rtt: string
+  loading: boolean
+}
+
+function getPingState(peerLatency: Record<string, PeerLatency>, peer: Peer): ReactElement {
```
Comment (Cafe137) [inline src/pages/peers/PeerTable.tsx:35]: Hmm, first I found it a bit confusing (why is it not `PingState`?), but I understand this is being used in the `map` function, and that is why you chose this function style?

Comment (vojtechsimetka) [reply src/pages/peers/PeerTable.tsx:35]: Yeah exactly. I was considering if this should be a component on its own but that seemed quite bizarre. For optimalization purposes it's defined outside of the PeerTable component as this function does not need to be recreated with each re-render.

**Review by Cafe137 — APPROVED** [review]
Thanks, LGTM 🙂

**Pull request merged**